docs/application-tutorial/xor_distance.md

```diff
@@ -0,0 +1,94 @@
+# Xor Distance
+
+We describe how to compute a XOR distance (as known as an Hamming weight distance) in Concrete. This
+can be useful in particular for biometry use-cases, where obviously, private is a very interesting
+feature.
+
+The full code can be done [here](../../frontends/concrete-python/examples/xor_distance/hamming_distance.py). Execution times of the different functions are given in the
+final section.
+
+## The Goal
+
+Goal here is to compute
+
+```
+def dist_in_clear(x, y):
+
+    return numpy.sum(hw(x ^ y))
```

Comment:
Maybe explain what `hw` is a bit?

Reply:
ok

Reply:
I think it's better to provide some input/output pairs to this function.

Reply:
ok, let me do that

Reply:
Adding:

+For example, for `x = 0xf3 = 0b11110011` and `y = 0xbc = 0x10111100`, we would have
+`x ^ y = 0b01001111 = 0x4f`, and so `dist_in_clear(x, y) = 5` since there are 5 set bits in `0x4f`.

Reply:
ok with you?

Reply:
Maybe the end could be:
```
since bits of `x` and bits of `y` differ in 5 different locations:
0b11110011
0x10111100
   ^  ^^^^
```

Reply:
ok I'll add it

Reply:
done a bit later